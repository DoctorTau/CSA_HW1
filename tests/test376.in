316
3195 -4287 -1666 -8655 6778 9991 -405 -1541 1786 9706 9241 5090 8326 -2161 -1528 -9506 6225 -8825 833 4284 0 3045 5087 -9303 203 7648 -6679 -7863 -5650 5967 9212 6787 -5392 -8268 5910 -1324 -2920 9348 -8354 -4212 -4478 -8572 5689 537 3510 -9575 2407 1087 -6517 8996 4665 -3994 6253 -6634 -8954 7607 2257 2514 -3581 204 -8125 8425 -3524 9692 5834 5851 4902 8455 2322 7610 -121 6913 -7836 4532 -2655 -9723 -3641 6940 -2073 6091 -9657 -5196 -8981 -1952 1233 9346 -2645 -3637 -1948 -7952 -1854 4575 5000 1045 -3206 -5748 1135 6785 5731 -3324 -8618 -6004 548 -1225 -5159 2170 -9468 3134 5489 -1728 4491 -6839 8909 9881 8753 -6007 8633 -8451 2812 8236 -7638 4249 1051 2956 5297 3544 -4908 -70 6541 -9771 -5942 -9959 702 2983 1163 -6892 3260 -1377 -4143 -6626 -962 -7051 2708 -1174 5344 6588 2982 -5726 -899 7630 -6884 995 2301 8474 5889 4723 1700 -7336 5900 -3045 -7975 6973 9867 9496 -9784 -5613 -8352 -8402 -873 -3622 567 3836 9623 -5118 1376 1400 5858 -9183 6073 -9902 2962 7204 -4534 -5357 -3693 -1522 9416 -7622 -8897 7141 8307 5196 -7099 -1493 5098 7805 -7275 -1898 4591 3864 -3762 359 5456 -8504 -1627 6834 -2100 -8710 -2198 1192 7722 2977 -3572 6823 7971 2258 5309 867 -5243 -4970 3175 3741 -630 -4409 8820 -1169 -3715 -55 -9305 1273 -5772 8691 -846 766 9439 4318 -6542 -9250 -6806 1621 -8960 5771 6350 4965 3824 4257 7431 8305 -1760 3490 2038 -2041 -6840 6371 -6005 5791 8131 -5927 802 -3597 -6914 -9686 537 2250 -1289 -6810 -3749 -9936 6720 -9895 -4970 4985 -9002 -7888 -7096 -1708 -5227 4389 9361 -644 -9054 81 -9777 2741 5662 4199 -8829 -7992 2733 -4586 -3468 420 -8019 6061 1580 9480 8624 8034 -6697 5951 7912 -3471 -1106 9234 9213 -3577 -8309 -767 9327 999 -8315 -1423 6475 2956 -9652 -5143 