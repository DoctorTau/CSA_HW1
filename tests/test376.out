-1666 -1528 -9506 -5650 -5392 -8268 -1324 -2920 -8354 -4212 -4478 -8572 -3994 -6634 -8954 -3524 -7836 -5196 -1952 -1948 -7952 -1854 -3206 -5748 -3324 -8618 -6004 -9468 -1728 -7638 -4908 -70 -5942 -6892 -6626 -962 -1174 -5726 -6884 -7336 -9784 -8352 -8402 -3622 -5118 -9902 -4534 -1522 -7622 -1898 -3762 -8504 -2100 -8710 -2198 -3572 -4970 -630 -5772 -846 -6542 -9250 -6806 -8960 -1760 -6840 -6914 -9686 -6810 -9936 -4970 -9002 -7888 -7096 -1708 -644 -9054 -7992 -4586 -3468 -1106 -9652 